Pipfile

```
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+sklearn = "*"
```

```suggestion
scikit-learn = "*"
```
According to the pypi docs - https://pypi.org/project/sklearn/